### PR TITLE
Remove facade assemblies from VSIX output

### LINKED ISSUE #940: Remove reference assemblies that are added to the vsix
List of files inside the package:

* catalog.json
* Esent.Interop.dll
* extension.vsixmanifest
* manifest.json
* Microsoft.AspNetCore.Razor.Evolution.dll
* Microsoft.CodeAnalysis.Elfie.dll
* Microsoft.CodeAnalysis.Razor.Workspaces.dll
* Microsoft.CodeAnalysis.Remote.Razor.dll
* Microsoft.VisualStudio.LanguageServices.Razor.dll
* Microsoft.VisualStudio.RazorExtension.dll
* Microsoft.VisualStudio.RazorExtension.pdb
* Microsoft.VisualStudio.RazorExtension.pkgdef
* razorLanguageService.servicehub.service.json
* System.AppContext.dll
* System.Console.dll
* System.Diagnostics.FileVersionInfo.dll
* System.Diagnostics.StackTrace.dll
* System.IO.Compression.dll
* System.IO.FileSystem.dll
* System.IO.FileSystem.Primitives.dll
* System.Runtime.InteropServices.RuntimeInformation.dll
* System.Security.Cryptography.Algorithms.dll
* System.Security.Cryptography.Encoding.dll
* System.Security.Cryptography.Primitives.dll
* System.Security.Cryptography.X509Certificates.dll
* System.Text.Encoding.CodePages.dll
* System.Threading.Thread.dll
* System.ValueTuple.dll
* System.Xml.ReaderWriter.dll
* System.Xml.XmlDocument.dll
* System.Xml.XPath.dll
* System.Xml.XPath.XDocument.dll
* [Content_Types].xml

Most of the System.* binaries are reference assemblies and should not be required to be included in the package.

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #940

### CODE REVIEW

**Comment by pranavkm** [comment]
What's in the VSIX now:

* extension.vsixmanifest
* Microsoft.CodeAnalysis.Remote.Razor.dll
* Microsoft.AspNetCore.Razor.Evolution.dll
* Microsoft.CodeAnalysis.Razor.dll
* Microsoft.CodeAnalysis.Razor.Workspaces.dll
* Microsoft.VisualStudio.LanguageServices.Razor.dll
* Microsoft.VisualStudio.RazorExtension.dll
* Microsoft.VisualStudio.RazorExtension.pdb
* Esent.Interop.dll
* Microsoft.CodeAnalysis.Elfie.dll
* System.IO.Compression.dll
* System.Runtime.InteropServices.RuntimeInformation.dll
* System.Security.Cryptography.Algorithms.dll
* System.ValueTuple.dll
* System.Xml.XPath.XDocument.dll
* Microsoft.VisualStudio.RazorExtension.pkgdef
* razorLanguageService.servicehub.service.json
* [Content_Types].xml
* manifest.json
* catalog.json

The 4 System.* binaries I left in there have some implementation

**Comment by Eilon** [comment]
So you did this by looking in the package and seeing what came from a `ref` folder and we're now excluding them?

I wonder if this is ever at risk of changing in the future?

**Comment by pranavkm** [comment]
It's probably pretty safe as long as we continue targeting the same TFM. It's super unlikely they introduce new APIs in released frameworks. I could make a note of it in where we define the TFM.

**Comment by Eilon** [comment]
Yeah would be good to have a note of that if/when we update the TFM in the future.

File: tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj [hunk 1]
```diff
@@ -27,6 +27,10 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.VisualStudio.RazorExtension</RootNamespace>
     <AssemblyName>Microsoft.VisualStudio.RazorExtension</AssemblyName>
+    <!--
+    We exclude some facade-only assemblies (i.e. assemblies without any implementation) from the VSIX using the SuppressFromVsix ItemGroup listed in this file.
```
Comment (pranavkm) [inline tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj]: cc @Eilon

**Pull request merged**